tools/azure-sdk-tools/devtools_testutils/azure_testcase.py

```diff
@@ -241,6 +256,11 @@ def create_basic_client(self, client_class, **kwargs):
             client.config.enable_http_logger = True
         return client
 
+    def create_basic_client(self, client_class, **kwargs):
+        """ DO NOT USE ME ANYMORE."""
```

Comment:
tiniest nit: do we want to log a warning instead of having a comment, more discoverable